.github/workflows/runSuperLinter.yml

```diff
@@ -13,13 +13,13 @@ jobs:
       statuses: write
     steps:
       - name: Check Out Code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           # Full git history is needed to get a proper list of changed files
           fetch-depth: 0
         # Lint changed files of these langauges using Super Linter shared action
       - name: Lint Code Base
-        uses: github/super-linter/slim@v5
+        uses: super-linter/super-linter@v6.0.0
```

Comment:
Awesome! Can you switch over to the `slim` variant if you have a chance? I found that it loads and runs MUCH faster on CI since it doesn't have to load a bunch of linters for languages we aren't using or worried about 

https://github.com/super-linter/super-linter?tab=readme-ov-file#super-linter-variants

Reply:
oh you did!  sorry the comments were outdated and I didn't notice. nice work ; we really appreciate your time and expertise here!